crates/holochain_diagnostics/src/lib.rs

```diff
@@ -50,7 +50,7 @@ where
 }
 
 pub fn config_standard() -> ConductorConfig {
-    holochain::sweettest::standard_config()
+    holochain::sweettest::standard_config().into()
```

Comment:
I changed the return type of `sweettest::standard_config()` into `SweetConductorConfig`, so now it needs an `.into()` to convert it into the `ConductorConfig`.

Reply:
Yes, I meant why do you prefer the return type to SweetConductorConfig and then intoing to ConductorConfig?

Reply:
@jost-s Ah! - Because this way, if you're working with sweettest, you now have the ability to slightly adjust the `standard_config()` vs having to recreate it all from scratch before you then freeze it into a `ConductorConfig`.

Reply:
💡